### ISSUE TITLE
Ubuntu 18.04 problem

### ISSUE THREAD

**Issue by EduardKononov** [yeongbin-jo/python-chromedriver-autoinstaller#1]
# 'By the way'
Before I start, want to say that the tool is **awesome**! It makes end users' life way easier. Thank you for publishing it.

# Issue
**System**: Ubuntu 18.04 LTS
**Python**: 3.8
**Chrome version**: 
```bash
$ google-chrome --version
Google Chrome 72.0.3626.119 
```

**Code snipped that repoduces the problem**:
```python
import chromedriver_autoinstaller
chromedriver_autoinstaller.install()
```  
**Trackeback**:
```
Traceback (most recent call last):
  File "/home/deadad/PycharmProjects/sticker_downloader/chrome.py", line 23, in <module>
    chromedriver_autoinstaller.install()
  File "/usr/local/lib/python3.6/dist-packages/chromedriver_autoinstaller/__init__.py", line 15, in install
    chromedriver_filepath = utils.download_chromedriver(cwd)
  File "/usr/local/lib/python3.6/dist-packages/chromedriver_autoinstaller/utils.py", line 161, in download_chromedriver
    chrome_version = get_chrome_version()
  File "/usr/local/lib/python3.6/dist-packages/chromedriver_autoinstaller/utils.py", line 102, in get_chrome_version
    with subprocess.Popen(['chromium-browser', '--version'], stdout=subprocess.PIPE) as proc:
  File "/usr/lib/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.6/subprocess.py", line 1364, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'chromium-browser': 'chromium-browser'
```

Seems like it tries to figure out my **chromium-browser** version, but I **do not have it install**. However, I used to things this way and it **did** work just perfect:
```python
from selenium import webdriver

driver_settings = {
    # downloaded manually from https://chromedriver.chromium.org/downloads
    'executable_path': 'drivers/chromedriver', 
}
with webdriver.Chrome(**driver_settings) as driver:
    ...
```
So the problem is not with the machine.

## Possible solution
My first assumptation is that in my case the lib has to check `google-chrome --version`, not `chromium-browser`'s one.

Thank in advance!

**Comment by dzaiats** [comment]
@EduardKononov , just install chromium-browser on your machine. I had the same issue. It is a workaround of course.
`apt install -y chromium-browser`

**Comment by yeongbin-jo** [comment]
Thank you for reporting.

I am currently working on merging multiple webdriver autoinstallers into one library.
When that is done, you will be able to use more stable functions. Test automation will also be introduced at that time.

I apologize for any inconvenience.

**Comment by yeongbin-jo** [comment]
And solved by #2
